src/connection-loop.ts

```diff
@@ -151,18 +151,18 @@ export class ConnectionLoop {
         let ok: boolean;
         try {
           lastSendTime = start;
-          log?.('Sending request');
+          debug?.('Sending request');
           ok = await delegate.invokeSend();
-          log?.('Send returned', ok);
+          debug?.('Send returned', ok);
         } catch (e) {
-          log?.('Send failed', e);
+          info?.('Send failed', e);
```

Comment:
Not sure if this counts as info given @phritz rubrick.

Reply:
Here is the proposal:

https://github.com/rocicorp/replicache-old/issues/35

- Establish three log levels:
  - ERROR: something truly unexpected has happened and a developer should go look.
  - INFO: an important change of state or information that is immediately useful to the developer.
  - DEBUG: verbose information about what's happening that might be useful to a developer.

Reply:
info seems OK to me here.